docs/shared/node-tutorial/1-code-generation.md

```diff
@@ -0,0 +1,136 @@
+---
+title: 'Node Nx Tutorial - Part 1: Code Generation'
+description: In this tutorial you'll create a backend-focused workspace with Nx.
+---
+
+{% callout type="check" title="Integrated Repo" %}
```

Comment:
I'd reword the callout. It should be something about two styles which point to gettingstarted/intro